db/tests/resources/filter_sort_create.sql

```diff
@@ -0,0 +1,131 @@
+CREATE TABLE "FilterSort" (
+    mathesar_id integer NOT NULL,
+    "varchar" varchar(32),
+    "numeric" integer,
+    "date"    date,
+    "array"   integer[]
+);
+
+CREATE SEQUENCE "filter_sort_mathesar_id_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
```

Comment:
Please see:  https://www.postgresql.org/docs/13/datatype-numeric.html#DATATYPE-SERIAL

Also, are you actually using this sequence?  It seems like there's lots of trouble involved in setting it up, but it's not actually used anywhere.